src/modules/recipe/components/RecipeActions/RecipeActions.styles.ts

```diff
@@ -0,0 +1,10 @@
+import styled from 'styled-components';
+
+import Button from '~recipe/ui/Button';
+
+export const RecipeActions = styled.div``;
```

Comment:
??

Reply:
Overall, I think I made it too complex. My idea of this exercise it was to make it in a "professional" way, even it was training. So, I believed that in a real component styles will take so much lines, so makes sense to exist in separate file. Same for types. This was my initial thought, but probably I should took it just as a training.